spring-cloud-context/src/main/java/org/springframework/cloud/context/named/NamedContextFactory.java

```diff
@@ -67,6 +67,12 @@ public void setConfigurations(List<C> configurations) {
 		return new HashSet<>(contexts.keySet());
 	}
 
+	protected void createAndCacheContexts() {
```

Comment:
Yes, still not thought of that @ryanjbaxter - it would have to reside in `spring-cloud-netflix` as it is to support this downstream issue https://github.com/spring-cloud/spring-cloud-netflix/issues/1334. I am guessing some kind of listener that calls this overridden method of `SpringClientFactory`